### PR TITLE
Add pg volume-expansion docs

### PR DESCRIPTION
**Pull request by takiaesha**

### CODE REVIEW

File: docs/examples/postgres/volume-expansion/pg-standalone.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: kubedb.com/v1alpha2
+kind: Postgres
+metadata:
+  name: pg-standalone
+  namespace: demo
+spec:
+  version: "13.13"
+  replicas: 3
```
Comment (souravbiswassanto) [inline docs/examples/postgres/volume-expansion/pg-standalone.yaml]: For standalone, replica should be `1`

File: docs/examples/postgres/volume-expansion/pg-ha-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: kubedb.com/v1alpha2
+kind: Postgres
+metadata:
+  name: pg-ha-cluster
+  namespace: demo
+spec:
+  version: "13.13"
+  replicas: 2
```
Comment (souravbiswassanto) [inline docs/examples/postgres/volume-expansion/pg-ha-cluster.yaml]: make `replicas: 3`

File: docs/examples/postgres/volume-expansion/pg-ha-cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+apiVersion: kubedb.com/v1alpha2
```
Comment (souravbiswassanto) [inline docs/examples/postgres/volume-expansion/pg-ha-cluster.yaml:1]: create a new file named `pg-cluster-with-arbiter.yaml` and provide replicas: 2 there.

File: docs/examples/postgres/volume-expansion/vol-exp-arbiter.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+apiVersion: ops.kubedb.com/v1alpha1
+kind: PostgresOpsRequest
+metadata:
+  name: pgops-vol-exp-arbiter
+  namespace: demo
+spec:
+  apply: IfReady
+  databaseRef:
+    name: pg-ha-cluster
```
Comment (souravbiswassanto) [inline docs/examples/postgres/volume-expansion/vol-exp-arbiter.yaml]: database name should be like this `pg-cluster-with-arbiter`. This name should be provided in ` pg-cluster-with-arbiter.yaml` file.

File: docs/examples/postgres/volume-expansion/vol-exp.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+apiVersion: ops.kubedb.com/v1alpha1
```
Comment (souravbiswassanto) [inline docs/examples/postgres/volume-expansion/vol-exp.yaml]: rename this file as vol-exp-standalone.yaml

File: docs/guides/elasticsearch/autoscaler/compute/overview/index.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 title: Elasticsearch Compute Autoscaling Overview
 menu:
   docs_{{ .version }}:
-    identifier: es-auto-scaling-overview
+    identifier: es-auto-scaling-Overview
```
Comment (ArnobKumarSaha) [inline docs/guides/elasticsearch/autoscaler/compute/overview/index.md]: Identifies should be small letter

File: docs/guides/mongodb/volume-expansion/replicaset.md [hunk 7]
```diff
@@ -135,6 +135,7 @@ spec:
   volumeExpansion:
     mode: "Online"
     replicaSet: 2Gi
+    mode: Online
```
Comment (ArnobKumarSaha) [inline docs/guides/mongodb/volume-expansion/replicaset.md:137]: `mode: Online` twice

File: docs/guides/mongodb/volume-expansion/standalone.md [hunk 8]
```diff
@@ -129,13 +129,17 @@ spec:
   volumeExpansion:
     mode: "Online"
     standalone: 2Gi
+    mode: Online
```
Comment (ArnobKumarSaha) [inline docs/guides/mongodb/volume-expansion/standalone.md:131]: same.  Find & remove for this type usages

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

File: docs/guides/proxysql/quickstart/mysqlgrp/index.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 title: Load Balance To MySQL Group Replication With KubeDB Provisioned ProxySQL
 menu:
   docs_{{ .version }}:
-    identifier: guides-proxysql-quickstart-overview
+    identifier: guides-proxysql-quickstart-Overview
```
Comment (souravbiswassanto) [inline docs/guides/proxysql/quickstart/mysqlgrp/index.md]: make lowercase

**Review by souravbiswassanto — CHANGES_REQUESTED** [review]

**Review by souravbiswassanto — APPROVED** [review]

**Pull request merged**